qutebrowser/config/configdata.yml

```
@@ -1769,6 +1769,14 @@ tabs.wrap:
   type: Bool
   desc: Wrap when changing tabs.
 
+tabs.focus_stack_size:
+  default: 10
+  type:
+    name: Int
+    minval: -1
+    maxval: maxint
+  desc: The maximum stack size to remember for tab switches (-1 for no maximum).
```

```suggestion
  desc: Maximum stack size to remember for tab switches (-1 for no maximum).
```

(see https://github.com/qutebrowser/qutebrowser/commit/56671e578707f05c8e580705ef4eab61c0069e93)